.github/workflows/daily.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       RPC_ENDPOINT_TESTNET: ${{ secrets.RPC_ENDPOINT_TESTNET }}
       RPC_ENDPOINT_MAINNET: ${{ secrets.RPC_ENDPOINT_MAINNET }}
     strategy:
+      max-parallel: 1
```

Comment:
We should discuss which number to pick, If we run everything sequentially, the whole workflow would take a long time (maybe more than a day). I'd increase max-parallel to something like 10.

Reply:
You are right, silly me. I'll leave it to 10 to discuss it.